package.json

```diff
@@ -38,11 +39,35 @@
         "category": "ansible"
       },
       {
-        "command": "vscode-ansible.playbook-in-terminal",
-        "title": "Run Ansible Playbook in Terminal",
+        "command": "vscode-ansible.playbook-in-docker",
+        "title": "Run Ansible Playbook in Docker",
+        "category": "ansible"
+      },
+      {
+        "command": "vscode-ansible.playbook-in-localansible",
+        "title": "Run Ansible Playbook in Local Ansible",
```

Comment:
while "in Docker" sounds ok,
i think "in Local Ansible" doesn't sound quite right.
Maybe should be "using local Ansible setup", or just "(local Ansible)" in brackets?

Reply:
just following the pattern 'in xxx' seems more organized. any other suggestion to make commands organized? in Local Ansible Setup?

Reply:
ok. maybe it should be like this
"Run Ansible Playbook"
"Run Ansible Playbook in Docker"
"Run Ansible Playbook in CloudShell"

We could just omit "in local Ansible" as it's a kind of obvious it's default option.

Reply:
"Run Ansible Playbook in Local Ansible" give user hint that they need have ansible installed locally to use this option